datahub-web-react/src/app/entityV2/shared/EntitySearchSelect/ActorsSearchSelect.tsx

```diff
@@ -0,0 +1,239 @@
+        display: flex;
+        align-items: center;
+    }
+`;
+
+const TitleContainer = styled.div`
+    display: flex;
+    flex-direction: column;
+`;
+
+export interface ActorsSearchSelectProps {
+    selectedActorUrns: string[];
+    onUpdate: (selectedActors: (CorpUser | CorpGroup)[]) => void;
+    placeholder?: string;
+    defaultActors?: (CorpUser | CorpGroup)[];
+    isDisabled?: boolean;
+    width?: number | 'full' | 'fit-content';
+    showSearch?: boolean;
+}
+
+/**
+ * A specialized search and selection component for actors (CorpUser and CorpGroup entities).
+ * Built on top of SimpleSelect with actor-specific features like recommendations,
+ * avatar rendering, and placeholder support.
+ *
+ * TODO: Support resolving selected entities from selectedActorUrns on initial render.
+ */
+export const ActorsSearchSelect: React.FC<ActorsSearchSelectProps> = ({
+    selectedActorUrns,
+    onUpdate,
+    placeholder = 'Search for users or groups',
+    defaultActors: placeholderActors,
+    isDisabled = false,
+    width = 'full',
+    showSearch = true,
+}) => {
+    const entityRegistry = useEntityRegistryV2();
+    const [selectedActorEntities, setSelectedActorEntities] = useState<Entity[]>([]);
+    const hasAutoSelectedRef = useRef(false);
+
+    // Auto-select placeholder actors ONLY ONCE on initial render when no actors are selected
+    useEffect(() => {
+        if (
+            placeholderActors &&
+            placeholderActors.length > 0 &&
+            !hasAutoSelectedRef.current &&
+            selectedActorUrns.length === 0
+        ) {
+            onUpdate(placeholderActors);
+            hasAutoSelectedRef.current = true;
+        }
+    }, [placeholderActors, selectedActorUrns.length, onUpdate]);
+
+    // Autocomplete query for actors (CorpUser and CorpGroup types)
+    const [autoCompleteQuery, { data: autocompleteData, loading: searchLoading }] =
+        useGetAutoCompleteMultipleResultsLazyQuery({
+            fetchPolicy: 'no-cache',
+        });
+
+    const { recommendedData, loading: recommendationsLoading } = useGetRecommendations([
+        EntityType.CorpGroup,
+        EntityType.CorpUser,
+    ]);
+
+    // Get results from the recommendations or autocomplete
+    const searchResults: Array<Entity> = useMemo(() => {
+        return (
+            autocompleteData?.autoCompleteForMultiple?.suggestions?.flatMap((suggestion) => suggestion.entities) ||
+            recommendedData ||
+            []
+        );
+    }, [autocompleteData?.autoCompleteForMultiple?.suggestions, recommendedData]);
+
+    // Sync selectedActorEntities with selectedActorUrns from parent
+    useEffect(() => {
+        const currentSelectedUrns = selectedActorEntities
+            .map((e) => e.urn)
+            .sort()
+            .join(',');
+        const newSelectedUrns = selectedActorUrns.sort().join(',');
+
+        // Only update if the URNs have actually changed
+        if (currentSelectedUrns !== newSelectedUrns) {
```

Comment:
i think this will always be true with lists. you could use lodash's `isEqual` here